packages/uniforms-bootstrap4/src/BoolField.tsx

```diff
@@ -32,14 +32,19 @@ function Bool({ onChange, ...props }: BoolFieldProps) {
     { ...props, label: labelBefore, value: props.value },
     <div
       className={classnames(inputClassName, 'form-check', 'checkbox', {
-        'text-danger': error,
         'custom-control-inline': inline,
+        'text-danger': error,
+        'text-success':
+          !props.error && props.value !== undefined && props.changed,
```

Comment:
Shouldn't this logic be consistent with the logic for adding the class name to the input?

Reply:
The `text-success` was added here accidentally - if we want to stay consistent with the 'error' style we should stylize only the label. Thanks for this.